lib/pluginmanager/command.rb

```diff
@@ -1,8 +1,16 @@
 class LogStash::PluginManager::Command < Clamp::Command
+
+  def initialize(invocation_path, context = {}, parent_attribute_values = {})
+    initialize_gemfile
+    super(invocation_path, context, parent_attribute_values)
+  end
+
```

Comment:
Ok, now I understand that this refactor was done to accommodate setting `Gem.sources` here. I don't think we should do this here, for the same reasons I gave about not setting `Gem.sources` globally. Setting `Gem.sources` definitely does not belong here. Also this refactor breaks the purpose of having a lazy `@gemfile`.